examples/plot_hf_hub.py

```diff
@@ -90,8 +90,10 @@
 # %%
 # Model Card
 # ==========
-# We will now create a model card and save it
-model_card = card.Card(model)
+# We will now create a model card and save it. For more information about how
+# to create a good model card, refer to the :ref:`model card example
+# <sphx_glr_auto_examples_plot_model_card.py>`
+model_card = card.Card(model, metadata=card.metadata_from_config(Path(local_repo)))
```

Comment:
As a user, I might be confused about where the config comes from. How about adding a comment above the `init` call that mentions that a `config.json` is created for the user, similar to the comment in `plot_model_card.py`?

Reply:
clarified!